src/node/utils.ts

```diff
@@ -17,23 +18,18 @@ export function routesToPaths(routes?: RouteRecordRaw[]) {
   const paths: Set<string> = new Set()
 
   const getPaths = (routes: RouteRecordRaw[], prefix = '') => {
-    // remove trailing slash
-    prefix = prefix.replace(/\/$/g, '')
     for (const route of routes) {
-      let path = route.path
-      // check for leading slash
-      if (route.path) {
```

Comment:
I guess in your case, all we need to do is change this line to `if (route.path != null)`